static/docs/commands-reference/fetch.md

```diff
@@ -1,62 +1,287 @@
+
+- `-j JOBS`, `--jobs JOBS` - number of threads to run simultaneously to handle
+  the downloading of files from the remote. Using more jobs may improve the
+  total download speed if a combination of small and large files are being
+  fetched. The default value is `4 * cpu_count()`. For SSH remotes default is 4.
+  > Note, this option applies only to fetching from local remotes or from SSH 
+  remotes. (See `dvc remote` for more information about remotes.)
+
+- `-a`, `--all-branches` - fetch cache for all branches, not just the active
+  one. This means that you'll the files needed to reproduce different versions
+  of a DVC file ([experiments](/doc/get-started/experiments)), not just the
+  current one.
+
+- `-T`, `--all-tags` - fetch cache for all tags. Similar to `-a` above
+
+- `--show-checksums` - show checksums instead of file names when printing the
+  download progress.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+- `-q`, `--quiet` - do not write anything to standard output.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
 
 ## Examples
 
-Fetch all files used in the current Git branch:
+To explore `dvc fetch` let's consider a simple workspace with several stages and
+a few Git tags. Then we can see what happens with `git` and `dvc fetch` as we
+shift from tag to tag.
+
+<details>
+
+### Click and expand to setup the project
+
+This step is optional, and you can run it only if you want to run this examples
+in your environment. First, you need to download the project:
+
+```shell
+    $ git clone https://github.com/iterative/example-get-started
+```
+
+Second, let's install the requirements. But before we do that, we **strongly**
+recommend creating a virtual environment with `virtualenv` or a similar tool:
+
+```shell
+    $ cd example-get-started
+    $ virtualenv -p python3 .env
+    $ source .env/bin/activate
+```
+
+Now, we can install requirements for the project, which include DVC with S3
+support:
+
+```shell
+    $ pip install -r requirements.txt
+```
+
+</details>
+
+The existing pipeline looks almost like in this
+[example](/doc/get-started/example-pipeline):
+
+```shell
+    .
+    ├── data
+    │   └── data.xml.dvc
+    ├── evaluate.dvc
+    ├── featurize.dvc
+    ├── prepare.dvc
+    ├── train.dvc
+    └── src
+        └── <code files here>
+```
+
+We have these tags in the repository that represent different iterations of
+solving the problem:
+
+```shell
+    $ git tag
+
+    baseline     <- first simple version of the model
+    bigram       <- use bigrams to improve the model
+```
+
+## Examples: Default behavior
+
+This project comes with a predefined S3 [remote
+storage](https://man.dvc.org/remote). We can now just run `dvc fetch`
+that will download the most recent `model.pkl`, `data.xml`, and other files that
+are under DVC control into our local cache:
+
+```shell
+    $ ldvc status -c    <- compares local cache vs default remote
```

Comment:
ldvc -> dvc

Reply:
let's remove the comment. It makes even harder to copy the command. Either do it #-style above the line, or explain command you run in text. Also, if we keep `dvc status` should we should some output at least? Does it makes sense to split it into two paragraphs - one with status + explanation, another one just fetch.

Reply:
Added output and removed the comment from the code block, explained in a note below the block.